module/extension/Director/src/provider/ProviderGroup.hpp

```diff
@@ -40,8 +40,6 @@ namespace module::extension::provider {
         enum State {
             /// This Provider group currently isn't doing anything
             IDLE,
-            /// This Provider group is currently in an "Entering" state
-            ENTERING,
```

Comment:
Can't be in an entering state?

Reply:
Entering was previously done so that it could run some warmup tasks before doing the normal providers. However, I realised while trying to code it there is no difference between an entering and a normal provider so it makes no sense to have entering.

The role of entering is now just handled by having a normal provider that does what you want and the ordering is now handled by the fact that the system will try to run the versions of the provider in the order that you declare them.

so if you have a normal provider for walk that needed standing it couldn't run, but if you had a second provider declared after that that would cause standing it'll do the warmup automatically without a need for a specific entering element

Reply:
Ah fair enough